packages/styles/scss/components/tabs-extended-media/_tabs-extended-media.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright IBM Corp. 2016, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@import '../../globals/imports';
+@import '../link-list/index';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/layout/breakpoint';
+@import 'carbon-components/scss/components/tabs/tabs';
+@import 'carbon-components/scss/components/accordion/accordion';
+@import 'carbon-components/scss/components/accordion/keyframes';
+@import '@carbon/ibmdotcom-styles/scss/themes/expressive/tokens';
+@import '@carbon/ibmdotcom-styles/scss/themes/expressive/components/tabs';
+@import '@carbon/ibmdotcom-styles/scss/themes/expressive/components/accordion';
+
+@mixin tabs-extended-media {
+  :host(#{$dds-prefix}-tabs-extended-media) {
+    margin: 0;
+    grid-column: 1 / span 16;
+    @include carbon--breakpoint(sm) {
+      padding-top: $carbon--spacing-07;
+      padding-bottom: $carbon--spacing-12;
+    }
+    @include carbon--breakpoint(lg) {
+      padding-top: $carbon--spacing-10;
+      padding-bottom: 10rem;
```

Comment:
Can use `$spacing-13` here.

Reply:
It doesn't look like `$carbon--spacing-13` exists.

Reply:
strange, there should be here: https://carbondesignsystem.com/guidelines/spacing/overview/

Reply:
@jeffchew it looks like it only goes up to `$carbon--spacing-13`. I just checked locally